evalai/utils/auth.py

```diff
@@ -76,6 +76,37 @@ def get_user_auth_token():
         sys.exit(1)
 
 
+def validate_user_auth_token_by_profile(auth_token):
+    """
+    Checks tokens availability via server.
+    """
+    url = "{}{}".format(get_host_url(), URLS.profile.value)
+    try:
+        headers = {"Authorization": "Token {}".format(auth_token)}
+        response = requests.post(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        if response.status_code in EVALAI_ERROR_CODES:
+            echo(
+                style(
+                    "Failed: Could not make a authentication with this token.",
```

Comment:
@takitsuse can you reframe this error message. This doesn't seem correct grammatically

Reply:
Ok...how about "Failed: Unable to validate token."?